code/index.html

```diff
@@ -1,22 +1,174 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+        <title>Project 2: The Wheeeekly News</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Yusei Magic">
+    <style>
+      body {
+        font-family: 'Yusei Magic', sans-serif;
+      }      
+      </style>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
-  <body>
+
+  <body class="container">
+
     <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+      <div class="left">
+        <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+        <div class="logo-group">
+        <div class="logo">
+          <img src="/assets/logo.png" alt="logo image">
+        </div>
+        <div class="nameplate">
+          <img src="/assets/title.png" alt="website name">
+        </div>
+      </div>
+      </div>
 
-    <section class="big-news">
-      <!-- The big news content -->
+      <div class="middle">
+        <div class="logo">
+          <img src="./assets/logo.png" alt="logo image">
```

Comment:
Let's try to be consistent about how we write our `<img src` paths
Either `./assets/...` or `/assets/...`